modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -365,11 +365,11 @@ def hlr_v2_response_schemas(ref_root)
     }
   end
 
-  def nod_create_schemas
```

Comment:
Don't we want to keep the v1 schemas around for production documentation?

Reply:
I'm confused about this. We didn't for hlr...but wait, hlr had the old schema set up. But also, don't we hand pick what we change?

Ugh. The rswag stuff is so annoying.